plugins/turing/runner/experiment_runner_test.go

```diff
@@ -0,0 +1,136 @@
+package runner
```

Comment:
I notice the integration test has not been ported over (which makes sense because we're not running the treatment service for this). Can we instead add a similar test into this file using appropriate mocking?